python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -109,18 +118,32 @@ def validate(self):
         if self.writeParquetOnly and not self.doWriteParquetTables:
             raise ValueError("Cannot writeParquetOnly if doWriteParquetTables is False")
 
+
 class CoaddAnalysisRunner(TaskRunner):
     @staticmethod
     def getTargetList(parsedCmd, **kwargs):
         kwargs["cosmos"] = parsedCmd.cosmos
 
         # Partition all inputs by tract,filter
         FilterRefsDict = functools.partial(defaultdict, list)  # Dict for filter-->dataRefs
-        tractFilterRefs = defaultdict(FilterRefsDict)  # tract-->filter-->dataRefs
-        for patchRef in sum(parsedCmd.id.refList, []):
-            tract = patchRef.dataId["tract"]
-            filterName = patchRef.dataId["filter"]
-            tractFilterRefs[tract][filterName].append(patchRef)
+        # Make sure the actual input files requested exist (i.e. do not follow the parent chain)
+        # First check for forced catalogs.  Break out of datasets loop if forced catalogs were found,
+        # otherwise continue search for existence of unforced catalogs
+        for dataset in ["forced_src", "meas"]:
+            tractFilterRefs = defaultdict(FilterRefsDict)  # tract-->filter-->dataRefs
+            for patchRef in sum(parsedCmd.id.refList, []):
+                tract = patchRef.dataId["tract"]
+                filterName = patchRef.dataId["filter"]
+                inputDataFile = patchRef.get("deepCoadd_" + dataset + "_filename")[0]
```

Comment:
Thank you. I forgot that this change was on this ticket.  Note that this works because the current butler doesn't check the registry for deepCoadds and multiband catalogs.